src/ParallelAlgorithms/Interpolation/Interpolate.hpp

```diff
@@ -51,11 +52,20 @@ void interpolate(
       });
 
   // Send volume data to the Interpolator, to trigger interpolation.
-  auto& interpolator = *Parallel::local_branch(
-      ::Parallel::get_parallel_component<Interpolator<Metavariables>>(cache));
-  Parallel::simple_action<Actions::InterpolatorReceiveVolumeData<
-      typename InterpolationTargetTag::temporal_id>>(
-      interpolator, temporal_id, array_index, mesh, interpolator_source_vars);
+  if (interpolator_id.has_value()) {
+    auto interpolator =
```

Comment:
Could you add a comment about what the `interpolator_id` is/represents? I think that'll be helpful for future people (me) editing this.